src/web/services/feedback-sessions.service.ts

```diff
@@ -202,13 +203,12 @@ export class FeedbackSessionsService {
   /**
    * Gets a set of givers that has given at least one response in the feedback session.
    */
-  getFeedbackSessionSubmittedGiverSet(
-      courseId: string, feedbackSessionName: string): Observable<FeedbackSessionSubmittedGiverSet> {
+  getFeedbackSessionSubmittedGiverSet(queryParams: { courseId: string, feedbackSessionName: string }):
+      Observable<FeedbackSessionSubmittedGiverSet> {
     const paramMap: { [key: string]: string } = {
-      courseid: courseId,
-      fsname: feedbackSessionName,
+      courseId: queryParams.courseId,
```

Comment:
should be in lowercase? `courseid`?

Please check for similar error. We are following the HTTP conversion that all params should be in lowercase.

Reply:
Sorry for the oversight, I've made the changes. This was the only `paramMap` in camelcase.